cpp/tensorrt_llm/kernels/cutlass_kernels/python/generate_kernels.py

```diff
@@ -97,13 +97,10 @@ def GetDataTypeBits(type):
     return DataTypeSize[type]
 
 
-def GetDataTypeNames(type, is_mx_fp4=None):
-    mxprefix = ""
-    if is_mx_fp4 is not None:
-        mxprefix = "mx_" if is_mx_fp4 else "nv_"
+def GetDataTypeNames(type):
```

Comment:
I would really like the MXFP4 stuff to be open sourced in the MR, but even if we don't we still need to be able to use MXFP4 from the internal repo